bip-emilengler-terminology-for-payment-recipient-identifiers.mediawiki

```diff
@@ -0,0 +1,54 @@
+<pre>
+  BIP: UNDEFINED
+  Title: Terminology for payment recipient identifiers
+  Author: Emil Engler <me@emilengler.com>, Marco Falke <example@example.com>
```

Comment:
This will need to be multiple lines.

You can add me too if you want, since I guess I ended up providing a bunch of content in the end. ^^;

Reply:
Sure thing, what E-Mail should I use?

Reply:
Name added, just need an E-Mail.

Reply:
Just my normal git email:

```
Author:     MarcoFalke <falke.marco@gmail.com>

Reply:
Done